test/Domain/LeanCode.DomainModels.MassTransitRelay.Tests/ScopedFiltersTests.cs

```diff
@@ -10,34 +10,44 @@
 
 namespace LeanCode.DomainModels.MassTransitRelay.Tests;
 
-public class ScopedFiltersTests : IAsyncLifetime
+public class ScopedFiltersTests : IAsyncLifetime, IDisposable
 {
     private readonly IBusControl bus;
     private readonly IBusActivityMonitor activityMonitor;
     private readonly Interceptor interceptor = new();
+    private readonly ServiceProvider serviceProvider;
 
     public ScopedFiltersTests()
     {
         var services = new ServiceCollection();
-        var builder = new ContainerBuilder();
 
-        var modules = new AppModule[] { new MassTransitModule(), new MassTransitTestRelayModule(), };
-
-        foreach (var m in modules)
-        {
-            m.ConfigureServices(services);
-            builder.RegisterModule(m);
-        }
-
-        builder.RegisterType<TestService>().AsSelf().InstancePerLifetimeScope();
-
-        builder.RegisterInstance(interceptor).AsSelf();
-
-        builder.Populate(services);
-        var container = builder.Build();
-        bus = container.Resolve<IBusControl>();
-
-        activityMonitor = container.Resolve<IBusActivityMonitor>();
+        services.AddMassTransit(cfg =>
+        {
+            cfg.AddConsumer(typeof(Consumer));
+
+            cfg.UsingInMemory(
+                (ctx, cfg) =>
+                {
+                    cfg.ReceiveEndpoint(
+                        "queue",
+                        rcv =>
+                        {
+                            Filter1Observer.UseFilter1(rcv, ctx);
+                            Filter2Observer.UseFilter2(rcv, ctx);
+
+                            rcv.ConfigureConsumers(ctx);
+                        }
+                    );
```

Comment:
Shouldn't we somehow rewrite that in an _endpoint per consumer_ way to promote good practices? Also that should be our main case so the test would be more relevant.